docs/scarpet/api/Auxiliary.md

```diff
@@ -4,36 +4,50 @@ Collection of other methods that control smaller, yet still important aspects of
 
 ## Sounds
 
+Functions which allow to manipulate the player's auditory environment.
+
 ### `sound()`, `sound(name, pos, volume?, pitch?, mixer?)`
 
 Plays a specific sound `name`, at block or position `pos`, with optional `volume` and modified `pitch`, and under
-optional `mixer`. Default values for `volume`, `pitch` and `mixer` are `1.0`, `1.0`, and `master`. 
+optional `mixer`. Default values for `volume`, `pitch` and `mixer` are `1.0`, `1.0`, and `master`.
 Valid mixer options are `master`, `music`, `record`, `weather`, `block`, `hostile`,`neutral`, `player`, `ambient`
-and `voice`. `pos` can be either a block, triple of coords, or a list of three numbers. Uses the same options as a
- corresponding `playsound` command.
- 
-Used with no arguments, return the list of available sound names.
- 
+and `voice`. `pos` can be either a block, triple of coords, or a list of thee numbers. Uses the same options as a
```

Comment:
It's the number 3. However, isn't "a list of three numbers" already mentioned by "triple of coords"?